javascript/javascript3/week3/canvasPart1_main.js

```diff
@@ -0,0 +1,35 @@
+//Lets make some art using classes
+const body = document.querySelector("body");
+const myCanvasToDrawCircle = document.getElementById("canvasId");
+const ctx = myCanvasToDrawCircle.getContext("2d");
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    ctx.beginPath();
```

Comment:
In constructor function you would want to set the instance (the new circle object) properties, and we keep the methods outside of the constructor. 

 ```

// with the constructor method you set out what arguments will be taken when creating a new object and you assign it to the properties of the future object you will be using in your methods. A class is basically a template for the objects.

constructor(x, y, r, startAngle, endAngle, fillColor)
{
        this.x =x 
        this.y = y
        this.radius = radius
        this.startAngle = startAngle
        this.endAngle = endAngle
        this.fillColor = fillColor
      }

```